ThingIFSDK/ThingIFSDK/TriggeredCommandForm.swift

```diff
@@ -47,32 +47,73 @@ open class TriggeredCommandForm: NSObject, NSCoding {
     /// Meta data of ad command.
     open let metadata: Dictionary<String, Any>?
 
+    /// Use trait or not.
+    open let useTrait: Bool
 
     // MARK: - Initializing TriggeredCommandForm instance.
+    init(actions: [[String : Any]],
+         useTrait: Bool,
```

Comment:
@yongpingchen 

if `useTrait` becomes optional, compiler can not distinguish `init(action:useTrait:title:commandDescription:metadata:)` and `init(action:title:commandDescription:metadata:)`

So that code can not be built.